mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -862,4 +874,7 @@ The new line here must be kept as the second half of the string is clickable for
     <!-- Sessions List -->
     <!-- Content description (not visible, for screen readers etc.): Button that closes a tab n the sessions list -->
     <string name="close_tab">Close tab</string>
+
+    <!-- CFR for Toolbar Shield Icon (Banner Info Message) -->
```

Comment:
Please give a little bit more context around what "CFR for Toolbar Shield icon" is or does, so that localizers understand more about where and why this string shows up (this also helps with their testing the string once it is localized)